app/parse_with_gemini.py

```
@@ -246,48 +268,114 @@ def parse_with_ids(
+                log.warning("⚠️ Ошибка создания обогащенного MD с заглушкой")
+        except Exception:
+            log.exception("❌ Ошибка создания обогащенного MD с заглушкой")
+    else:
+        log.info("ℹ️ AI будет использоваться - обогащенный MD будет создан после получения AI результатов")
 
     return db_id, lot_ids_map, processed_data
 
 
+def _regenerate_reports_for_lot(tender_id: str, lot_id: str, ai_results: Dict[str, Any]):
+    """
+    Регенерирует MD и chunks файлы для лота с реальными AI данными.
```

Ruff (0.14.2)

Docstring contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF002)